wmf/handlers.go

```diff
@@ -2259,17 +2282,20 @@ func (self *Handler) Static(resp http.ResponseWriter, req *http.Request) {
 	/* This should be handled by something like an nginx rule
 	 */
 
+	self.SecureHeaders(resp)
 	if !self.config.GetFlag("use_insecure_static") {
 		return
 	}
-
-	http.ServeFile(resp, req, self.docRoot+req.URL.Path)
+	filePath := path.Clean(self.docRoot + req.URL.Path)
```

Comment:
thus why the option was "use_insecure_static", but it's a bit less insecure now.